pkgs/os-specific/linux/vtune-sepdk/default.nix

```diff
@@ -0,0 +1,106 @@
+{
+  bash,
+  coreutils,
+  gnugrep,
+  gnused,
+  intel-oneapi-vtune,
+  kernel,
+  kmod,
+  lib,
+  makeWrapper,
+  p7zip,
+  procps,
+  python3,
+  shadow,
+  stdenv,
+  su,
+  which,
+}:
+
+let
+  binPath = lib.makeBinPath [
+    bash
+    coreutils
+    gnugrep
+    gnused
+    kmod
+    procps
+    python3
+    shadow
+    su
```

Comment:
that is useless here as it does not have the setuid bit in the nix store. We nee to rely on the wrapper.